src/utils/stringifiers.ts

```diff
@@ -0,0 +1,61 @@
+import type { BasicType, Type } from '../interfaces.js';
+
+export function defaultStringify(valueType: BasicType | 'mixed', value: unknown, name: string): string | undefined {
```

Comment:
This function could use a bit of jsDoc. Especially the behavior for object/array/mixed types.